guides/security/data_protection.md

```diff
@@ -0,0 +1,27 @@
+# Data Protection
+
+Made Tech complies with regulations under the European General Data Protection Regulation (GDPR) and Data Protection Act 2018. It is the company’s policy to protect and hold safe all Personal Information which it requires or needs to possess in the performance and management of its operation.
+
+Under GDPR, an individual has the right to be informed whether or not their personal data is being processed and, if so, to be informed of the following:
+ - What personal data is being processed 
+ - The purposes for which the personal data is being processed
+ - Who, if anyone, the personal data is disclosed to
+ - The extent to which the personal data is being used to make automated decisions relating to the individual
+
+Below are guides for team members on the course of action to take if they receive a request regarding personal data.
+
+## Made Tech Data
+
+For Made Tech data (e.g. requests from Made Tech employees about personal data), please forward any requests from your team on to the Operations team who will process it thusly:
+ - We will acknowledge receipt of the data subject’s request.
+ - We will ask for two proofs of identity to confirm requester ID. 
+ - We will follow up and ask for information on the types of data in question (e.g. type and timeframe)
+ - We will process the request accordingly and respond to the data subject in line with the legislation. 
+ - We will inform the individual that a record of their basic information (e.g. name and company) may be kept with a note to not contact again.
+ - We may ask for input and/or provision of data from other teams to comply with the request. 
+
+## Customer and Partner Data
+
+Where a customer or partner makes a request about their data, seek written approval from the that they have verified the request and authorise Made Tech to collate and release this data from systems we have access to. They may also need to provide proof of ID for the requester. 
+
+Once this is done, refer the request to the Operations Team and we will process the request in line with the process outlined above.
```

Comment:
I read this section slightly differently. To me this was if we're being asked to extract data from a customers system about one of their customers. e.g. a Teacher comes to DfE and asks for all the data the department have on them.

If this is the case I think the team will need to execute the request, but should seek reasonable certainty that the request has been validated by the customer.